src/main/java/de/tum/in/www1/artemis/domain/text/TextPairwiseDistance.java

```diff
@@ -0,0 +1,64 @@
+package de.tum.in.www1.artemis.domain.text;
+
+import javax.persistence.*;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import de.tum.in.www1.artemis.domain.DomainObject;
+
+/**
+ *  Pairwise distance between two TextBlocks
+ */
+@Entity
+@Table(name = "text_pairwise_distance")
```

Comment:
the following annotations are missing

@Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
@JsonInclude(JsonInclude.Include.NON_EMPTY)